pkg/types/report.go

```diff
@@ -29,11 +29,12 @@ type Metadata struct {
 	OS   *ftypes.OS `json:",omitempty"`
 
 	// Container image
-	ImageID     string        `json:",omitempty"`
-	DiffIDs     []string      `json:",omitempty"`
-	RepoTags    []string      `json:",omitempty"`
-	RepoDigests []string      `json:",omitempty"`
-	ImageConfig v1.ConfigFile `json:",omitempty"`
+	ImageID        string                `json:",omitempty"`
+	DiffIDs        []string              `json:",omitempty"`
+	RepoTags       []string              `json:",omitempty"`
+	RepoDigests    []string              `json:",omitempty"`
+	ImageConfig    v1.ConfigFile         `json:",omitempty"`
+	LayersMetadata ftypes.LayersMetadata `json:",omitempty"`
```

Comment:
It belongs to `Metadata`. `Layers` is descriptive enough for me.

```suggestion
	Layers ftypes.LayersMetadata `json:",omitempty"`
```

Reply:
updated in https://github.com/aquasecurity/trivy/pull/8394/commits/fb65b083f1ce928e702868b49727bce1233a62d6